root/etc/sudoers.d/50_nsapi

```diff
@@ -215,4 +215,5 @@ Cmnd_Alias NSAPI_ADMINS = \
     NSAPI_SYSTEM_TERMINAL, \
     NSAPI_SYSTEM_STORAGE, \
     NSAPI_SYSTEM_USERS_GROUPS, \
+    NSAPI_SYSTEM_APPS, \
     /usr/libexec/nethserver/api/system-apps/update
```

Comment:
Ok to fix the warning. Please note that the line 219 has some magic: when an user can run `system-apps/update` he's granted `isAdmin` permission at UI level too.